### PR TITLE
Features section

### PR DESCRIPTION
**Pull request by sauloperez**
This another iteration of the landing page design we rolled out with the matchfunding campaing. This now aims to fill the gap many people have when trying to understand what Katuma is. We've been asked *¿Cuál es la función de Katuma exactamente?* quite often.

As we always say, Katuma is not just a digital platform, so I'm afraid we'll have to add a page soon explaining that other side.

### CODE REVIEW

**Comment by sauloperez** [comment]
screencapture-localhost-8080-features-html-2018-03-28-19_24_31

**Comment by sauloperez** [comment]
I might add more whitespace between features :thinking:

File: features.html [hunk 1]
```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
+  <head>
+      <meta charset="utf-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+      <title>Katuma</title>
+      <meta name="description" content="">
```
Comment (andresgutgon) [inline features.html]: Empty tag

File: features.html [hunk 2]
```diff
@@ -1,215 +1,80 @@
...
-      });
-    }
-
-  </script>
-  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
-  </body>
-</html>
+    </aside>
+  </div>
+</div>
```
Comment (andresgutgon) [inline features.html]: OMG you level up 💪 

But Jekyll is not cool anymore. You could use  https://www.gatsbyjs.org/ it use React and GraphQL 💯

Comment (sauloperez) [reply features.html]: *ayparfavar* @andresgutgon :trollface:

**Comment by sauloperez** [comment]
screencapture-localhost-4000-features-html-2018-04-05-12_55_15

Done

**Comment by andresgutgon** [comment]
Deploy! 👏

**Pull request merged**